ts/webui/src/components/experiment/trialdetail/chart/DefaultMetricPoint.tsx

```diff
@@ -86,7 +90,7 @@ class DefaultPoint extends React.Component<DefaultPointProps, DefaultPointState>
                     <div class="tooldetailAccuracy">
                         <div>Trial No.: ${data.data[0]}</div>
                         <div>Trial ID: ${data.data[2]}</div>
-                        <div>Default metric: ${data.data[1]}</div>
+                        <div>${userSelectAccuracyNumberKey} metric: ${data.data[1]}</div>
```

Comment:
If user has selected a key, there's no need to add a suffix `metric`.

Reply:
so default metric will show `default: xxx`